helpers/init_centos.sh

```diff
@@ -0,0 +1,209 @@
+#!/bin/bash
```

Comment:
Do we need this? I think it can be dropped.

Reply:
I think this can be dropped. I dont see it being used in code

Reply:
Done